packages/console/src/components/CodeEditor/index.tsx

```diff
@@ -33,12 +33,20 @@ const CodeEditor = ({ language, isReadonly = false, value = '', onChange }: Prop
           onChange={handleChange}
         />
         <SyntaxHighlighter
+          wrapLongLines
+          codeTagProps={{
+            style: {
+              fontFamily: "'Roboto Mono', monospace",
+            },
```

Comment:
can it use `className` for less style-in-code?

Reply:
Unfortunately, no. The `react-syntax-highlighter` will generate a `pre` tag and a `code` element inside the `pre`. Both of them have inline styles by default, so I can only use inline styles to override it.

Reply:
Most of the styles on `pre` tag can be inherited into the inner `code`. However, `code` has its own default font-family style `monospace`. So I have to specificly override this one to keep `code` consistent with `pre`

Reply:
i see. would you like to add some comments for explaining these 2 conversations in code?

Reply:
Sure, np

Reply:
Added comments